package.json

```diff
@@ -25,35 +27,41 @@
   },
   "dependencies": {
     "dotenv": "^16.4.5",
+    "express": "^4.19.2",
     "process": "^0.11.10",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "react-router-dom": "^6.25.1",
     "sass": "^1.77.8",
     "terser-webpack-plugin": "^5.3.10",
     "url": "^0.11.3",
     "webpack-bundle-analyzer": "^4.10.2"
   },
   "devDependencies": {
-    "@babel/core": "7.24.7",
-    "@babel/preset-env": "7.24.7",
+    "@babel/core": "^7.24.7",
+    "@babel/preset-env": "^7.24.7",
     "@babel/preset-react": "^7.24.7",
     "@babel/preset-typescript": "^7.22.5",
-    "@types/node": "20.11.0",
-    "@types/react": "18.2.48",
-    "@types/react-dom": "18.2.18",
-    "babel-loader": "9.1.3",
+    "@svgr/webpack": "^8.1.0",
+    "@types/node": "^20.11.0",
+    "@types/react": "^18.2.48",
+    "@types/react-dom": "^18.2.18",
+    "@types/react-router-dom": "^6.25.1",
```

Comment:
Please fix this dependency version. On building it locally, there is error: 
![Screenshot 2024-07-23 at 12 13 53](https://github.com/user-attachments/assets/fe98ac00-fda9-4606-b4a0-1c8d8aa4c19f)

so please find correct dependency version and use it.